inst/config/link.yml

```diff
@@ -4,6 +4,14 @@ default:
     logo: "https://github.com/2DegreesInvesting/tiltWebTool/raw/main/inst/logo/logo.png"
     link: "https://www.tiltsmes.org/"
 
+  welcome:
+    text: "feedback here"
+    link: "https://docs.google.com/forms/d/1cwPCtlW3xP97aNIZ9Isge2zw5nJ-JnXvteckCO4oawo/viewform?edit_requested=true"
+
+  methodology:
+    text: "methodology here"
+    link: "https://docs.google.com/document/d/14syERZ-t1nzZaiotL77zcvKVqoJ_47NA/edit?usp=drive_web&ouid=115108949015890448832&rtpof=true"
```

Comment:
This URL isn't public. Is that intenentional?

![image](https://github.com/user-attachments/assets/5ae13605-7d7f-4f5b-bba0-0315cd31efdd)

Reply:
@maurolepore yes for now we would leave it as it is as the paper is not ready yet. I will make it publish once it is done. For now it is fine with a link but if you see issue here: https://github.com/2DegreesInvesting/tiltWebTool/issues/197 it would be nice to actually upload a PDF. I was wondering how complicated this would be and if you could make an estimation (is it something that you know straight away of implementing OR is it something that is unknown for you). But for now all fine with the link and the PDF comes after other issues are solved.

Reply:
With regard were the link should stand. Unfortunately, I don't know where I can see my pull requests. But I changed the methodology text here. 

The text now should be: METHODOLOGY
This paper introduces a methodology to derive climate metrics for SMEs,We present two indicators: a “relative emission indicator,” based on product emission intensity, and a “transition risk indicator,” assessing risks from the low-carbon transition. These indicators, built using Life Cycle Assessment (LCA) and climate scenario data, are tested on a database of 250,000 firms across five European countries.
Our method is reproducible, transparent, and scalable, requiring minimal data. It can be used to explore SME low-carbon transformation, climate risks, or inform financial decision-making.
Please find our methodology here. For further information and how to use the research data, please watch these two online webinars.

I thought that this code here: 
`methodology:`
    `text: "methodology here"`
defines that in the methodolgy section if there is the text `methodolgy here` that the link would be embedded. That's how I understood it. That means that looking at the new text the link would be at the bold phrase: 

The text now should be: METHODOLOGY
This paper introduces a methodology to derive climate metrics for SMEs,We present two indicators: a “relative emission indicator,” based on product emission intensity, and a “transition risk indicator,” assessing risks from the low-carbon transition. These indicators, built using Life Cycle Assessment (LCA) and climate scenario data, are tested on a database of 250,000 firms across five European countries.
Our method is reproducible, transparent, and scalable, requiring minimal data. It can be used to explore SME low-carbon transformation, climate risks, or inform financial decision-making.
Please find our **methodology here.** For further information and how to use the research data, please watch these two online webinars.

Reply:
ml01: I think lower case is fine.